### PR TITLE
[REF] *: remove legacy Markup

### PR DESCRIPTION
**Pull request by aab-odoo**
Use owl.markup instead.

Part of task~3439226

Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/hr_recruitment/static/src/js/tours/hr_recruitment.js [hunk 1]
```diff
@@ -1,7 +1,6 @@
 /** @odoo-module **/
 
 import { _t } from "@web/core/l10n/translation";
-import { Markup } from "@web/legacy/js/core/utils";
 import { registry } from "@web/core/registry";
 import { stepUtils } from "@web_tour/tour_service/tour_utils";
```
Comment (ged-odoo) [inline addons/hr_recruitment/static/src/js/tours/hr_recruitment.js:6]: funny that both markup were imported

**Comment by ged-odoo** [comment]
robodoo r+

**Comment by ged-odoo** [comment]
@robodoo override=ci/security 
this is basically only using the markup from owl everywhere instead of the old markup function for qweb

**Comment by aab-odoo** [comment]
robodoo r+ (rebased)

**Comment by aab-odoo** [comment]
robodoo r+ (typo ...)

**Pull request closed without merging**